dotcom-rendering/src/web/components/CricketScoreboard.tsx

```diff
@@ -107,21 +107,21 @@ export const CricketInnings = ({
 	// There can be up to 2 innings per team in a cricket match
 	switch (teamInnings.length) {
 		case 1:
-			return (
+			return teamInnings[0] ? (
```

Comment:
This is somewhat redundant, as we are already checking the length of the array.

This class of refactors could remove the switch pattern entirely, but would probably need to start with the longest option first.

Reply:
Alternative:

```ts
	// There can be up to 2 innings per team in a cricket match
	const [one, two] = teamInnings;
	if (one && two) {
		return (
			<p>
				{cricketScore({ innings: one, short: true })}
				{' & '}
				{cricketScore({ innings: two })} ({two.overs} overs)
			</p>
		);
	}

	if (one) {
		return (
			<p>
				{cricketScore({ innings: one })} ({one.overs} overs)
			</p>
		);
	}

	return <p>Yet to bat</p>;
```

Reply:
Opted for a `isTuple` helper.